roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -256,67 +276,39 @@ def add_two_photon_series(imaging, nwbfile, metadata, num_chunks=10):
         Auxiliary static method for nwbextractor.
         Adds two photon series from imaging object as TwoPhotonSeries to nwbfile object.
         """
-        if 'Ophys' not in metadata:
-            metadata['Ophys'] = {}
-
-        if 'TwoPthotonSeries' not in metadata['Ophys']:
-            metadata['Ophys']['TwoPhotonSeries'] = [{'name': 'TwoPhotonSeries',
-                                                     'description': 'optical_series_description'}]
+        metadata = dict_recursive_update(get_default_nwb_metadata(), metadata)
+        metadata = update_dict(metadata, NwbImagingExtractor.get_nwb_metadata(imaging))
         # Tests if ElectricalSeries already exists in acquisition
         nwb_es_names = [ac for ac in nwbfile.acquisition]
         opts = metadata['Ophys']['TwoPhotonSeries'][0]
         if opts['name'] not in nwb_es_names:
             # retrieve device
             device = nwbfile.devices[list(nwbfile.devices.keys())[0]]
-
-            # create optical channel
-            if 'OpticalChannel' not in metadata['Ophys']:
-                metadata['Ophys']['OpticalChannel'] = [{'name': 'OpticalChannel',
-                                                        'description': 'no description',
-                                                        'emission_lambda': np.nan}]
-
-            optical_channel = OpticalChannel(**metadata['Ophys']['OpticalChannel'][0])
-            # sampling rate
-            rate = float(imaging.get_sampling_frequency())
-
-            if 'ImagingPlane' not in metadata['Ophys']:
-                metadata['Ophys']['ImagingPlane'] = [{'name': 'ImagingPlane',
-                                                      'description': 'no description',
-                                                      'excitation_lambda': np.nan,
-                                                      'indicator': 'unknown',
-                                                      'location': 'unknown'}]
-            imaging_meta = {'optical_channel': optical_channel,
-                            'imaging_rate': rate,
-                            'device': device}
-            metadata['Ophys']['ImagingPlane'][0] = update_dict(metadata['Ophys']['ImagingPlane'][0], imaging_meta)
+            metadata['Ophys']['ImagingPlane'][0]['optical_channel'] = \
```

Comment:
updating metadata with a default dictionary of metadata (`get_default_nwb_metadata()`) helps in removing some lines of code here